services/order.py

```diff
@@ -0,0 +1,28 @@
+from db.models import MovieSession, Order, Ticket
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(
+    tickets: list[dict], username: str, date: str | None = None
+) -> None:
+    user = get_user_model().objects.get(username=username)

```

Comment:
Consider handling the `DoesNotExist` exception for `get_user_model().objects.get(username=username)` to avoid runtime errors if the user with the given username does not exist.